src/config.rs

```diff
@@ -0,0 +1,57 @@
+use actix_web::{fs, App};
+use std::net::IpAddr;
+
+use crate::auth;
+use crate::listing;
+
+#[derive(Clone, Debug)]
+pub struct MiniserveConfig {
+    pub verbose: bool,
+    pub path: std::path::PathBuf,
+    pub port: u16,
+    pub interfaces: Vec<IpAddr>,
+    pub auth: Option<auth::BasicAuthParams>,
+    pub path_explicitly_chosen: bool,
+    pub no_symlinks: bool,
+    pub random_route: Option<String>,
+    pub sort_method: listing::SortingMethods,
+    pub reverse_sort: bool,
+}
+
+pub fn configure_app(app: App<MiniserveConfig>) -> App<MiniserveConfig> {
```

Comment:
I think this particular function should stay in main.rs because main.rs essentially configures the whole actix app, right?

Reply:
Sure, I can do that. In this case, I think I will remove config.rs, otherwise it would just contain the `MiniserveConfig` struct (which will I would put back in main.rs), what do you think ?